hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileDeleteTest.java

```diff
@@ -158,7 +158,7 @@ void preHandleWorksAsExpected() throws PreCheckException {
         subject.preHandle(realPreContext);
 
         assertTrue(realPreContext.requiredNonPayerKeys().size() > 0);
-        assertEquals(3, realPreContext.requiredNonPayerKeys().size());
+        assertEquals(1, realPreContext.requiredNonPayerKeys().size());
```

Comment:
Might want to add assertions here that the 1 required key is a `Threshold` key with threshold `1`.

Reply:
sure will be done